homeassistant/components/overkiz/diagnostics.py

```diff
@@ -0,0 +1,19 @@
+"""Provides diagnostics for Overkiz."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from . import HomeAssistantOverkizData
+from .const import DOMAIN
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant, entry: ConfigEntry
+) -> list[dict]:
```

Comment:
We don’t need a type for the dict?

Reply:
Our setup reply is hard to type, however I have made the typing a bit more strict!
`list[dict[str, Any]]`